supervisor/shared/filesystem.c

```diff
@@ -71,7 +71,7 @@ static void make_empty_file(FATFS *fatfs, const char *path) {
 
 
 static void make_sample_code_file(FATFS *fatfs) {
-    #if CIRCUITPY_FULL_BUILD
+    #if 1
```

Comment:
Did you want to keep this, or can the whole if block be removed?

Reply:
oops, that was a testing thing. It's only 75 bytes to add a non-empty code.py, but that can be a separate PR.